.github/actions/private-tangle/setup/action.yml

```diff
@@ -40,6 +38,6 @@ runs:
     - name: Wait for tangle to start
       shell: bash
       run: wget -qO- https://raw.githubusercontent.com/eficode/wait-for/v2.2.4/wait-for | sh -s -- -t 120 http://localhost:8050/health -- echo "Tangle is up"
-    - name: Wait for faucet to start
-      shell: bash
-      run: wget -qO- https://raw.githubusercontent.com/eficode/wait-for/v2.2.4/wait-for | sh -s -- -t 120 http://localhost:8088/health -- echo "Faucet is up"
+    # - name: Wait for faucet to start
```

Comment:
?

Reply:
Faucet is currently broken as it's not update to the latest iota-core API changes (the ones changed in this PR :trollface: )

Reply:
Add a todo then?

Reply:
done